### ISSUE TITLE
Doc misunderstanding

### ISSUE THREAD

**Issue by Ptit-Biscuit** [OSLC/developing-oslc-applications#8]
Hi !
I've just started to work on an OSLC app and we use OAuth request / access tokens.
I was a bit confuse how the flows work so I try to google it and came across your doc [here](http://oslc.github.io/developing-oslc-applications/iotp_adaptor/authentication.html) and it says: 

> A consumer key/secret pair is essentially a login account for an application. If a functional user is associated with the consumer key, consuming apps can use an OAuth Authorization header with the consumer key and secret to access protected resources using One-legged OAuth.

Until here all is fine but later you have a schema (it's much easier to understand so thanks !) that describes how three-legged OAuth works (at least according to [the OAuth Bible](http://oauthbible.com/#oauth-10a-three-legged)).
I think it can misleads people and because it's not a easy subject maybe it needs to be improved a bit.

Thanks anyway :)

**Comment by jadelkhoury** [comment]
Thanks for the feedback @Ptit-Biscuit.

I found OAuth 1 quite hard to follow, and the Bible is indeed a good source.
Do you have to use OAuth1? Otherwise, I can recommend you adopt OAuth 2. Easier to implement, and you get much better library support.

**Comment by Ptit-Biscuit** [comment]
I'll try OAuth 2. From what I have found OAuth 1 is indeed harder to understand.

Thanks.

**Comment by jadelkhoury** [comment]
Good luck! 
For what it's worth, I currently dealing with a scenario where the OSLC consumer speakes OAuth1, but the server needs to use OAuth2 (since Oauth1 is not supported within the organsiation). 
no idea how to deal with the situation, since oauth1 & 2 are not compatible. 
But can see what I come up with, in case the problem is similar.
